src/Stripe.net/Services/BankAccounts/BankAccountService.cs

```diff
@@ -40,22 +41,22 @@ public virtual Task<BankAccount> CreateAsync(string parentId, BankAccountCreateO
 
         public virtual BankAccount Delete(string parentId, string id, BankAccountDeleteOptions options = null, RequestOptions requestOptions = null)
         {
-            return this.Request<BankAccount>(BaseAddress.Api, HttpMethod.Delete, $"/v1/customers/{parentId}/sources/{id}", options, requestOptions);
+            return this.Request<BankAccount>(BaseAddress.Api, HttpMethod.Delete, $"/v1/customers/{parentId}/sources/{WebUtility.UrlEncode(id)}", options, requestOptions);
```

Comment:
Wouldn't `parentId` also need to be escaped?

Reply:
Yes it definitely does.  Let me take another pass and make sure I escape all these.  Good catch!

Reply:
Ok, escaped parentId along with other named ids in other files.